recipes/file_read_backwards/recipe.yaml

```diff
@@ -0,0 +1,48 @@
+schema_version: 1
+
+context:
+  version: 3.2.0
+
+package:
+  name: file-read-backwards
+  version: ${{ version }}
+
+source:
+  - url: https://pypi.org/packages/source/f/file-read-backwards/file_read_backwards-${{
```

Comment:
With a single source, you can drop the dash and unindent `url` and `sha256`

Reply:
Done!